### ISSUE TITLE
Clippy 0.2.2 breaks Quill

### ISSUE THREAD

**Issue by zifeo** [softwaremill/scala-clippy#8]
Related to #6, when using both the Clippy plugin and Quill, the compilation is broken. Easily reproducible with any of [those examples](http://getquill.io).

```sbt
[error] ~/project/src/main/scala/Main.scala:17: exception during macro expansion: java.lang.StackOverflowError
[error]   db.run(query)(CSVSources.data.flatMap(_.toOption))
```

**Comment by mikolak-net** [comment]
This problem occurs whenever there's a compiler reporting call with on the `info` level, due to `scala.tools.nsc.reporters.Reporter.info` and `com.softwaremill.clippy.DelegatingReporter.info0` infinitely recursing into one another.

**Comment by adamw** [comment]
Released as 0.2.3

**Comment by zifeo** [comment]
Thanks!
